### PR TITLE
chore: Regenerate fresh package-lock.json files

### PR DESCRIPTION
**Pull request by dshekhtman**
I deleted all the `package-lock.json` files and ran `npm install` in all the projects to regenerate them to the latest versions.

### CODE REVIEW

File: contact-duplicate/src/app/app.functions/package-lock.json [hunk 1]
```diff
@@ -30,22 +29,19 @@
...
       "version": "0.4.0",
-      "resolved": "https://registry.npmjs.org/asynckit/-/asynckit-0.4.0.tgz",
-      "integrity": "sha512-Oei9OH4tRh0YqU3GxhX79dM/mwVgvbZJaSNaRk+bshkj0S5cfHcgYakreBjrHwatXKbz+IoIdYLxrKim2MjW0Q=="
+      "license": "MIT"
     },
     "node_modules/axios": {
       "version": "0.27.2",
-      "resolved": "https://registry.npmjs.org/axios/-/axios-0.27.2.tgz",
-      "integrity": "sha512-t+yRIyySRTp/wua5xEr+z1q60QmLq8ABsS5O9Me1AsE5dfKqgnCFzwiCZZ/cGNd1lq4/7akDWMxdhVlucjmnOQ==",
+      "license": "MIT",
```
Comment (banderson) [inline contact-duplicate/src/app/app.functions/package-lock.json]: Love to see it

File: contact-duplicate/src/app/extensions/package-lock.json [hunk 2]
```diff
@@ -102,294 +54,6 @@
...
-      "optional": true,
-      "os": [
-        "win32"
-      ],
-      "engines": {
-        "node": ">=12"
-      }
-    },
     "node_modules/@hubspot/ui-extensions": {
       "version": "0.0.1-beta.1",
```
Comment (banderson) [inline contact-duplicate/src/app/extensions/package-lock.json]: Looks like the update didn't take @dshekhtman

File: get-started/src/app/extensions/package-lock.json [hunk 3]
```diff
@@ -102,294 +54,6 @@
...
-      "optional": true,
-      "os": [
-        "win32"
-      ],
-      "engines": {
-        "node": ">=12"
-      }
-    },
     "node_modules/@hubspot/ui-extensions": {
       "version": "0.0.1-prealpha.7",
```
Comment (dshekhtman) [inline get-started/src/app/extensions/package-lock.json]: Here's another place that didn't take and I'm not sure why (yet).

File: contact-duplicate/src/app/extensions/package-lock.json (hunk 2 above)
Comment (dshekhtman) [reply contact-duplicate/src/app/extensions/package-lock.json]: Okay, I think I've sorted it. `node_modules` need to be deleted as well as `package-lock.json`. I've rerun everything.

File: get-started/src/app/extensions/package-lock.json (hunk 3 above)
Comment (dshekhtman) [reply get-started/src/app/extensions/package-lock.json]: Sorted it: https://github.com/HubSpot/ui-extensions-react-examples/pull/11#discussion_r1212083942

**Review by Randyjp — APPROVED** [review]

**Pull request merged**